runtime/experiments/concurrency/latency.gnuplot

```diff
@@ -13,7 +13,9 @@ set yrange [0:]
 
 set style histogram columnstacked
 
-plot 'latency.dat' using 1:2 title 'p50', \
-     'latency.dat' using 1:3 title 'p90', \
-     'latency.dat' using 1:4 title 'p99', \
-     'latency.dat' using 1:5 title 'p100', \
+plot 'latency.dat' using 1:8 title 'p100', \
```

Comment:
Would love to see the output of this. Would also be nice at some point in the future to have an actual CDF; not worth working on this now.

Reply:
I attempted to port this to some of the other experiments since they now produce a consistent output via the utility functions, but the way this is written seems to depend on the x axis being a numeric value. Based on what's checked into this repo, I don't think I got far enough with gnuplot to figure out how to have a set of strings on the x axis. I imagine some sort of google search of "gnuplot candlestick" would be useful.